bHY6yh-de8398/cod-bxxqqq_bHY6yh-de8398.xml

```diff
@@ -0,0 +1,163 @@
+      <editorialDecl>
+        <p>Encoding of this text has followed the recommendations of the LombardPress 1.0.0
+          guidelines for a diplomatic edition.</p>
+      </editorialDecl>
+    </encodingDesc>
+    <revisionDesc status="draft">
+      <listChange>
+        <change when="2023-07-04" status="draft" n="0.0.0">
+          <p>Created file for the first time.</p>
+        </change>
+      </listChange>
+    </revisionDesc>
+  </teiHeader>
+  <text xml:lang="la">
+    <front>
+      <div xml:id="starts-on">
+        <pb ed="#B" n="14-v"/>
+        <cb ed="#B" n="b"/>
+      </div>
+    </front>
+    <body>
+      <div xml:id="bHY6yh-de8398"><!-- p1l1-r29 -->
+        <head xml:id="bHY6yh-de8398-Hd1e108">Rubrica 29</head>
+        <head xml:id="bHY6yh-de8398-Hd1e111" type="question-title">De diabolo.</head>
+        <p xml:id="bHY6yh-de8398-d1e114">
+          De dyabolo Rubrica 29
+        </p>
+        <p xml:id="bHY6yh-de8398-d1e119">
+          <lb ed="#B"/>Plato in thimeo <quote xml:id="bHY6yh-de8398-Qd1e123">Invisibilium di<lb ed="#B" break="no"/>uinarum 
+          potestatum que</quote> 
+        </p>
+        <p xml:id="bHY6yh-de8398-d1e129">
+          <quote xml:id="bHY6yh-de8398-Qd1e133">demones 
+          <lb ed="#B"/>nuncupantur prestare rationem
+          <lb ed="#B"/>maius est opus quam ferre valeat 
+          <lb ed="#B"/>hominis ingenium</quote>
+        </p>
+        <p xml:id="bHY6yh-de8398-d1e142">
+          <ref xml:id="bHY6yh-de8398-Rd1e145"><sic>Esus</sic> filius sydrach 
+          <lb ed="#B"/>in ecclesiastico capitulo 21</ref> <quote xml:id="bHY6yh-de8398-Qd1e152">dum maledicit ipsius 
+          <lb ed="#B"/>dyabolum maledicit ipse animam suam</quote>
+        </p>
+        <p xml:id="bHY6yh-de8398-d1e158">
+          [omitted]
+        </p>
+        <p xml:id="bHY6yh-de8398-d1e168">
+          <lb ed="#B"/><ref xml:id="bHY6yh-de8398-Rd1e170">Iohannes in ewangelio capitulo 8</ref> <quote xml:id="bHY6yh-de8398-Qd1e172">dyabolus est
+          <lb ed="#B"/>mendax et pater eius</quote>
+        </p>
+        <p xml:id="bHY6yh-de8398-d1e180">
+          ¶ <ref xml:id="bHY6yh-de8398-Rd1e182">Idem epistola 
+          <lb ed="#B"/>capitulo 3</ref> <quote xml:id="bHY6yh-de8398-Qd1e182">Qui facit peccatum ex diabolo est 
+          <lb ed="#B"/>¶ In hoc apparuit filius dei ut dis<lb ed="#B" break="no"/>soluat 
+          opera diaboli</quote>
+        </p>
+        <p>
+          ¶ Idem in apocalypsi
+        <pb ed="#B" n="15-r"/>
+        <cb ed="#B" n="a"/>
+          <lb ed="#B"/>capitulo 12 Serpens antiqus qui vo<lb ed="#B" break="no"/>catur
+          daybolus et sathanas qui
+          <lb ed="#B"/>succedit in <sic>vnisisum</sic> orbem pro<lb ed="#B" break="no"/>iecta
+          est de celo in terram et ange<lb ed="#B" break="no"/>li
+          eius cum eo misi sunt
+        </p>
```

Comment:
just checking. This is an entry that your manuscript has that was not in the Venice transcription, correct. 

If so you did this correctly (though in the future you can also mark the boundaries of the `<ref>` and `<quote>`)